Doc/distributing/index.rst

```diff
@@ -35,11 +35,11 @@ Key terms
   repository of open source licensed packages made available for use by
   other Python users
 * the `Python Packaging Authority
-  <https://www.pypa.io/>`__ are the group of
-  developers and documentation authors responsible for the maintenance and
-  evolution of the standard packaging tools and the associated metadata and
-  file format standards. They maintain a variety of tools, documentation
-  and issue trackers on both `GitHub <https://github.com/pypa>`__ and
+  <https://www.pypa.io/>`__ are the group of developers and documentation
```

Comment:
No need to reformat this and lose the git blame.

Reply:
@merwok Thanks for the heads up. I've reverted that change.